.pre-commit-config.yaml

```diff
@@ -201,6 +201,18 @@ repos:
           - eslint-plugin-promise@6.1.1
           - eslint-plugin-vue@8.7.1
 
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v3.0.3
+    hooks:
+      - id: prettier
+        name: prettier
+        entry: prettier
+        files: ^client/
```

Comment:
We have js & ts files in `bindings`, I recommend to include those.

Reply:
I think it's all boilerplate. And it would mean that
- either I add configuration related to the bindings in `client/.prettierignore` which is weird
- put the prettier config directly into parsec-cloud's root folder.
Both can be done but I'd rather use prettier only for the client for now and change it later if we really need it.